config/config/sc-config.yaml

```diff
@@ -427,6 +427,10 @@ prometheus:
       enabled: false
       percent: 80
       sizeQuotaGB: 1000
+    totalSize:
```

Comment:
I suggest we add a comment explaining the difference between `s3BucketAlerts.size` and `s3BucketAlerts.totalSize`, what do you think?